plutus-ledger-api/test-plugin/Spec/Value.hs

```diff
@@ -2,7 +2,6 @@
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE TemplateHaskell   #-}
 
-{-# OPTIONS_GHC -fplugin PlutusTx.Plugin #-}
```

Comment:
Nothing to do with this PR: I'm just taking the opportunity to remove it while I'm here.

Reply:
Nope, I'm going to revert that.  It may be causing problems with at least one GHC version.